src/rdma_van.h

```diff
@@ -125,26 +126,39 @@ class RDMAVan : public Van {
     struct sockaddr_in addr;
     memset(&addr, 0, sizeof(addr));    
 
+    int af = PF_INET;
```

Comment:
why not AF_INET?

Reply:
it doesn't work, https://arnold.byted.org/#/job/6091/task/489981/trial/1574330